From fdf1d3097177a3d55b5aaba22dc66b5538c57471 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:25:32 -0400
Subject: [tools] perf: Fix redirection printouts

Message-id: <1408462094-14194-164-git-send-email-jolsa@redhat.com>
Patchwork-id: 88136
O-Subject: [PATCH RHEL7.1 BZ1131394 163/325] perf tools: Fix redirection printouts
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit cee972c0e6940ec75bab2d02f37e96d06ce143eb
Author: Ingo Molnar <mingo@kernel.org>
Date: Thu Oct 10 07:42:56 2013 +0200

description
===========
Fix the duplicate util/util printout Arnaldo reported:

       $ make V=1 O=/tmp/build/perf -C tools/perf/ util/srcline.o
   ...
       # Redirected target util/srcline.o =>     /tmp/build/perf/util/util/srcline.o
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/Makefile.perf b/tools/perf/Makefile.perf
index 2badb08..8bc6d0c 100644
--- a/tools/perf/Makefile.perf
+++ b/tools/perf/Makefile.perf
@@ -600,11 +600,11 @@ ifneq ($(OUTPUT),)
 %.o: $(OUTPUT)%.o
  @echo "    # Redirected target $@ => $(OUTPUT)$@"
 util/%.o: $(OUTPUT)util/%.o
- @echo "    # Redirected target $@ => $(OUTPUT)util/$@"
+ @echo "    # Redirected target $@ => $(OUTPUT)$@"
 bench/%.o: $(OUTPUT)bench/%.o
- @echo "    # Redirected target $@ => $(OUTPUT)bench/$@"
+ @echo "    # Redirected target $@ => $(OUTPUT)$@"
 tests/%.o: $(OUTPUT)tests/%.o
- @echo "    # Redirected target $@ => $(OUTPUT)tests/$@"
+ @echo "    # Redirected target $@ => $(OUTPUT)$@"
 endif
 
 # These two need to be here so that when O= is not used they take precedence
-- 
1.7.1